nucypher/blockchain/eth/actors.py

```diff
@@ -248,39 +258,65 @@ def deploy_user_escrow(self, allocation_registry: AllocationRegistry):
         self.user_escrow_deployers[principal_address] = user_escrow_deployer
         return user_escrow_deployer
 
-    def deploy_network_contracts(self,
-                                 staker_secret: str,
-                                 policy_secret: str,
-                                 adjudicator_secret: str,
-                                 user_escrow_proxy_secret: str,
-                                 ) -> Tuple[dict, dict]:
-        """
-        Musketeers, if you will; Deploy the "big three" contracts to the blockchain.
-        """
-
-        token_txs, token_deployer = self.deploy_contract(contract_name='NuCypherToken')
-        staking_txs, staking_deployer = self.deploy_contract(contract_name='StakingEscrow', plaintext_secret=staker_secret)
-        policy_txs, policy_deployer = self.deploy_contract(contract_name='PolicyManager', plaintext_secret=policy_secret)
-        adjudicator_txs, adjudicator_deployer = self.deploy_contract(contract_name='Adjudicator', plaintext_secret=adjudicator_secret)
-        user_escrow_proxy_txs, user_escrow_proxy_deployer = self.deploy_contract(contract_name='UserEscrowProxy', plaintext_secret=user_escrow_proxy_secret)
-
-        deployers = (token_deployer,
-                     staking_deployer,
-                     policy_deployer,
-                     adjudicator_deployer,
-                     user_escrow_proxy_deployer,
-                     )
-
-        txhashes = {
-            NucypherTokenDeployer.contract_name: token_txs,
-            StakingEscrowDeployer.contract_name: staking_txs,
-            PolicyManagerDeployer.contract_name: policy_txs,
-            AdjudicatorDeployer.contract_name: adjudicator_txs,
-            UserEscrowProxyDeployer.contract_name: user_escrow_proxy_txs,
-        }
-
-        deployers = {deployer.contract_name: deployer for deployer in deployers}
-        return txhashes, deployers
+    def deploy_network_contracts(self, secrets: 'Secrets', interactive: bool = True) -> dict:
+
+        deployment_receipts = dict()
+        gas_limit = 8_000_000  # TODO: Gas management
+
+        # NuCypherToken
+        token_receipts, token_deployer = self.deploy_contract(contract_name=NucypherTokenDeployer.contract_name,
+                                                              gas_limit=gas_limit)
+        paint_contract_deployment(contract_name=NucypherTokenDeployer.contract_name,
+                                  receipts=token_receipts,
+                                  contract_address=token_deployer.contract_address)
+        deployment_receipts[NucypherTokenDeployer.contract_name] = token_receipts
```

Comment:
So, the key space of the dict will be values of `NucypherTokenDeployer.contract_name`... does this value mutate then?  It's not obvious how to think about this.

Reply:
No - they do not mutate - they are hard-coded class attributes on the Deployer claas.  This is an effort to have a single source of contract names in the codebase.  In general, this method can be code-optimized somewhat with a loop and/or function.

Reply:
Maybe uhhh... :thinking:... Maybe use a constant?  `NucypherTokenDeployer.contract_name` is a hard thing for the reader to resolve in terms of understanding the keyspace here.

Reply:
Yeah I can see it that way.  You propose we have a constants module with all the contract name strings defined?

Reply:
:heavy_check_mark: